src/amplitude-wrapper.js

```diff
@@ -143,7 +143,43 @@ import { version } from '../package.json';
       };
   };
 
+  var pageViewPlugin = () => {
```

Comment:
I would prefer this to be a plugin package in the the SDK so that it can be versioned, tested, and potentially used by other non-GTM customers if they want it.